settings.gradle.kts

```diff
@@ -8,6 +8,5 @@ pluginManagement {
         kotlin("jvm") version kotlinVersion
         kotlin("plugin.serialization") version kotlinVersion
         id("org.jlleitschuh.gradle.ktlint") version ktlintVersion
-        id("maven-publish")
```

Comment:
blir ikke denne brukt build gradle?

Reply:
Jo, men settings brukes bare for å sette versjonen til en plugin. Det er en begrensning i `build.gradle.kts` at man ikke kan bruke versonsnumrene fra `gradle.properties` for plugins, så da gjør vi det her. `maven-publish` er fredeles med i plugins i `build.gradle.kts`.

(Så kan du spørre deg hvorfor vi ikke trenger versjonsnummer for `maven-publish`. Det aner jeg ikke.)